clientupdate/clientupdate.go

```diff
@@ -778,14 +777,6 @@ func (up *Updater) cleanupOldDownloads(glob string) {
 	}
 }
 
-func (up *Updater) downloadURLToFile(pathSrc, fileDst string) (ret error) {
```

Comment:
I think this is called on Windows:
https://github.com/tailscale/tailscale/blob/main/clientupdate/clientupdate_windows.go#L129-L131